src/components/bulk-email-tool/bulk-email-form/bulk-email-recipient/BulkEmailRecipient.jsx

```diff
@@ -50,18 +55,25 @@ export default function BulkEmailRecipient(props) {
             description="A selectable choice from a list of potential email recipients"
           />
         </Form.Checkbox>
-        <Form.Checkbox
-          key="track:verified"
-          value="track:verified"
-          disabled={selectedGroups.find((group) => group === DEFAULT_GROUPS.ALL_LEARNERS)}
-          className="col col-lg-4 col-sm-6 col-12"
-        >
-          <FormattedMessage
-            id="bulk.email.form.recipients.verified"
-            defaultMessage="Learners in the verified certificate track"
-            description="A selectable choice from a list of potential email recipients"
-          />
-        </Form.Checkbox>
+        {
+          // additional modes
+          courseModes
+          && courseModes.length > 1
+          && courseModes.map((courseMode) => (
```

Comment:
I propose to move logical operations into a separate variable, this will simplify the readability of the code.

```suggestion
const isCourseModes = courseModes && courseModes.length > 1;
...
isCourseModes && courseModes.map((courseMode) => (
```

Reply:
Good point! I already moved it. Thx!